### ISSUE TITLE
[axe-linter] axe-linter.yml isn't respecting the `exclude` option

### ISSUE THREAD

**Issue by professorplumb** [dequelabs/axe-core#4343]
### Product

axe Linter

### Product Version

4.8.1

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
An `axe-linter.yml` configuration file for a VSCode project with the axe-linter extension installed will exclude files and filepatterns listed under the `exclude` option from being linted.

#### Actual
No matter how I configure the files under `exclude`, they seem to be linted regardless under my setup.

#### How to Reproduce
I created a dummy project with the following two files in an otherwise empty directory under /tmp/:
`test.html`
```
<html>
  <body>
    <img src="http://www.example.com/image.png" />
  </body>
</html>
```

`axe-linter.yml`
```
rules:
  html-has-lang: false
exclude:
  - test.html
 ```

`test.html` has two deliberate violations: `html-has-lang` and `image-alt`. When I open this directory as a VSCode project with axe Accessibility Linter v4.8.1 installed, and open the `test.html` file, I see the violation for `image-alt` called out by the axe Accessibility Linter extension.

I am confident that it is picking up and parsing the `axe-linter.yml` configuration file, because the `html-has-lang` violation is correctly ignored by the `rules` configuration option. 

However, I've tried many ways of writing the entries under `exclude`, and I haven't been able to get it to exclude entire files from linting. I'm entirely open to the possibility that I'm missing something obvious in terms of how to configure this, but I can't think of what it would be.

#### Additional context
Other formats for the `exclude` option I've tried (non-exhaustive):
```
exclude:
  - "test.html"
  - 'test.html'
  - *.html # parsing error
  - "*.html"
  - '*.html'
  - test/**/* # after moving test.html into a test/ directory
  "test.html": true # parsing error
  "*.html": true # parsing error
```

Also:
```
exclude: test.html
#---
exclude: "*.html"
#---
exclude: test/**/* # this is how axe-linter.yml is configured in the axe-core repo
```

I've tried uninstalling and reinstalling the axe Accessibility Linter VSCode extension. Any suggestions are welcome!

**Comment by michael-siek** [comment]
Hey @professorplumb have you tried
```yml
exclude:
  - "**/test.html"
```

**Comment by professorplumb** [comment]
Apparently I hadn't, because that worked! Thanks @michael-siek!
